app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  
+  def create
```

Comment:
Added tests for the sessions controller. Please review them and provide feedback.